typescript/cognito-idp/functions/user-delete.ts

```diff
@@ -0,0 +1,65 @@
+import { APIGatewayEvent } from 'aws-lambda';
+import { Handler } from './handler';
+import * as AWS from 'aws-sdk';
+import { APIEventResponse } from './handler';
```

Comment:
Rename the `functions` directory to `lambda` (that has been our convention so far, see cdkworkshop.com as an example)

Reply:
Addressed these comments in my latest commit.